devolutions-gateway/src/api/webapp.rs

```diff
@@ -248,6 +248,7 @@ pub(crate) enum SessionTokenContentType {
         /// Unique ID for this session
         session_id: Uuid,
     },
+    NetScan {},
```

Comment:
style: Define unit-like variant when no fields are required (idiom)
```suggestion
    NetScan,
```